script/debug/loop_junit.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import shlex
+import subprocess
+
+DIR_CURR = os.path.dirname(os.path.realpath(__file__))
+DIR_SCRIPT = os.path.dirname(DIR_CURR)
+DIR_TESTING = os.path.join(DIR_SCRIPT, "testing")
+DIR_JUNIT = os.path.join(DIR_TESTING, "junit")
+FILE_RUN_JUNIT = os.path.join(DIR_JUNIT, "run_junit.py")
+LOG_FILE = os.path.join(DIR_CURR, "log_loop_junit.log")
+CMD = "python3 {} --build-type=debug --query-mode=simple".format(
+    FILE_RUN_JUNIT)
+
+if __name__ == "__main__":
```

Comment:
What is the purpose of this script? Add documentation / comments.

Reply:
for troubleshooting the `connection refused` issue in docker. i will remove it together with the entire `debug` folder